wp-includes/class-wp-web-app-manifest.php

```diff
@@ -59,9 +59,30 @@ public function init() {
 	 * Mainly copied from Jetpack_PWA_Manifest::render_manifest_link().
 	 */
 	public function manifest_link_and_meta() {
+		$manifest = $this->get_manifest();
 		?>
 		<link rel="manifest" href="<?php echo esc_url( rest_url( self::REST_NAMESPACE . self::REST_ROUTE ) ); ?>">
-		<meta name="theme-color" content="<?php echo esc_attr( $this->get_theme_color() ); ?>">
+		<meta name="theme-color" content="<?php echo esc_attr( $manifest['theme_color'] ); ?>">
+		<meta name="apple-mobile-web-app-capable" content="yes">
+		<meta name="mobile-web-app-capable" content="yes">
+		<meta name="apple-touch-fullscreen" content="YES">
+		<?php if ( ! empty( $manifest['icons'] ) ) : ?>
+			<?php
+			$icons = $manifest['icons'];
+			usort(
+				$icons,
+				function( $a, $b ) {
```

Comment:
The use of a closure is causing a syntax error in PHP 5.2. Refactoring it into a class method would fix the problem. I gave the closure as more of an example code not not necessarily for it to be used verbatim. Nevertheless, we are also talking about updating the minimum required PHP version to 5.6 (#124), at which point this would be unnecessary.